dataservice/api/common/enumerations.yaml

```diff
@@ -0,0 +1,165 @@
+## ------------ NOTES ----------- ##
+# 'Unknown' - contributor specified this in source data
+# 'Not Reported' - should be used as the default if no value is specified in
+# the column or if this column doesn't exist in source data
+
+## --------- Questions --------- ##
+# How should values be cased - lower case, proper/title case?
+# What should default value be if Not Reported in source data?
+
+study:
+    release_status:
```

Comment:
I need clarification on what release status is for. If something isn't released, then it will be released on a particular date/schedule according to contract/tos/whatever. No?